src/verify/hooks.ts

```diff
@@ -50,6 +52,10 @@ export function useVerifyPhoneNumber(phoneNumber: string, countryCallingCode: st
 
   const dispatch = useDispatch()
   const address = useSelector(walletAddressSelector)
+  const privateDataEncryptionKey = useSelector(dataEncryptionKeySelector)
+  const publicDataEncryptionKey = privateDataEncryptionKey
+    ? compressedPubKey(hexToBuffer(privateDataEncryptionKey))
+    : null
```

Comment:
nit: we should do this only when needed before calling the API instead of for each call of this hook.